src/main/webapp/app/shared/link-preview/services/linkify.service.ts

```diff
@@ -29,13 +31,16 @@ export class LinkifyService {
             const start = match.index;
             const end = start + url.length;
 
-            const linkableItem = {
+            // Check if url is wrapped in <> tags
+            const isRemoved = text[start - 1] === '<' && text[end - 1] === '>';
```

Comment:
Does this cause problems when the url is at the very front of the text? I.e. an indexOutOfBounds.
Additionally, why are you subtracting 1 from the end, shouldn't it be one step to the right and therefor add one?

Reply:
It does not cause any problems when the url is at the very front of the text.

Reply:
> why are you subtracting 1 from the end, shouldn't it be one step to the right and therefor add one?

actually it should be `end` because the `end` is the index right after the url and thats where the `>` is located.